API/src/main/java/de/brandwatch/minianalytics/api/controller/MentionController.java

```diff
@@ -2,24 +2,17 @@
 
 
 import com.google.gson.Gson;
-import de.brandwatch.minianalytics.api.solr.model.Mention;
-import de.brandwatch.minianalytics.api.solr.repository.MentionRepository;
+import de.brandwatch.minianalytics.api.service.MentionService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-
-import java.time.Instant;
-import java.time.LocalDateTime;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
-import java.time.temporal.TemporalAccessor;
-import java.util.List;
+import org.springframework.web.server.ResponseStatusException;
 
 @RestController
 public class MentionController {
```

Comment:
I don't think the logger of this class is used. But even if, it uses the wrong class (RestController)